app/serializers/email_serializer.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+class EmailSerializer < ActiveModel::Serializer
+  attribute :email
+  attribute :effective_at
+
+  def id
+    nil
+  end
+
+  # Returns the email address nested in the given object.
+  #
+  # @return [String] Email address.  Sample `object.email_address`:
+  #   {
+  #     "effective_date" => "2018-02-27T14:41:32.283Z",
+  #     "value" => "test2@test1.net"
+  #   }
+  #
+  def email
+    object&.email_address&.dig 'value'
+  end
+
+  # Returns the email's effective datetime nested in the given object
+  #
+  # @return [String] Effective datetime in the yyyy-MM-dd'T'HH:mm:ss format.
+  #   Sample `object.email_address`:
+  #     {
+  #       "effective_date" => "2018-02-27T14:41:32.283Z",
+  #       "value" => "test2@test1.net"
+  #     }
+  #
+  def effective_at
+    object&.email_address&.dig 'effective_date'
+  end
+end
```

Comment:
Doesn't look like you validate the values `object&.email_address&.dig 'effective_date'` & `object&.email_address&.dig 'effective_date'` that come back from EVSS - what if they are `nil` such as in the event EVSS alters their API?

Maybe the larger philosophical question: since this API is just a pass through, how much validation should we really do?

Reply:
Good point.

@jkassemi @kreek @omgitsbillryan - what are your thoughts on this globally, and for this PR?

Reply:
it depends 😄 but most of the time I'd say validation is for when you expect there to be an invalid or missing attribute, and for the majority of the APIs we integrate with we're not expecting invalid data.

Reply:
Agree. I think that's worth revisiting after we get a sense of the quality of the underlying data. 

It could be nice to have a validation pattern that we use, and maybe explicitly mark fields as "unvalidated"? This would require some thought, and I do feel like the LoE is high if we're doing this for each integration. Maybe a v2?